docs/Project.toml

```
@@ -32,7 +32,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 AmplNLWriter = "1"
-Documenter = "0.26, 0.27"
+Documenter = "0.26, 0.27, 1"
```

The `make.jl` scripts are not backwards compatible, so this would be more appropriate:

```suggestion
Documenter = "1"
```